src/components/Scorecards/ScorecardDetailsPage/ScorecardRuleExemptionsTab/ScorecardRuleExemptionsTab.tsx

```diff
@@ -14,25 +14,21 @@
  * limitations under the License.
  */
 import React, { useMemo } from 'react';
-import {
-  Rule,
-  Scorecard,
-  ScorecardRuleExemptionResult,
-  ScorecardServiceScore,
-} from '../../../../api/types';
-import { Box, Tooltip, Typography, makeStyles } from '@material-ui/core';
+import { Rule, Scorecard, ScorecardRuleExemptionResult, ScorecardServiceScore, } from '../../../../api/types';
+import { Box, makeStyles, Tooltip, Typography } from '@material-ui/core';
 import { getRuleTitle } from '../../../../utils/ScorecardRules';
 import { RuleExemptionStatus } from './RuleExemptionStatus';
 import { CortexInfoCard } from '../../../Common/CortexInfoCard';
-import {
-  getExpirationText,
-  getIsActive,
-} from './ScorecardRuleExemptionsTabUtils';
+import { getExpirationText, getIsActive, } from './ScorecardRuleExemptionsTabUtils';
 import ScorecardRuleExemptionTooltip from './ScorecardRuleExemptionTooltip';
 import { ScorecardServiceRefLink } from '../../../ScorecardServiceRefLink';
 import { isEmpty } from 'lodash';
+import { StringIndexable } from '../../../ReportsPage/HeatmapPage/HeatmapUtils';
+import { HomepageEntity } from '../../../../api/userInsightTypes';
+import { entityComponentRef } from '../../../../utils/ComponentUtils';
 
 interface ScorecardRuleExemptionsTabProps {
+  entitiesByTag: StringIndexable<HomepageEntity>;
```

Comment:
nit: would prefer to not propagate usage of `StringIndexable`
```suggestion
  entitiesByTag: Record<string, HomepageEntity>;
```